foreign_cc/utils.bzl

```diff
@@ -44,7 +44,7 @@ def runnable_binary(name, binary, foreign_cc_target, match_binary_name = False,
 
     wrapper_cmd = """
     sed s@EXECUTABLE@$(rlocationpath {name})@g $(location @rules_foreign_cc//foreign_cc/private:runnable_binary_wrapper.sh) > tmp
-    sed s@SH_BINARY_FILENAME@{sh_binary_filename}@g tmp > $@
```

Comment:
note[nonblocking]: I was wondering about this, so I checked on the history.  `SH_BINARY_FILENAME` was added in 41c937a7a94e0c20bd19d02983d0e8c4f760fcde and the last (and only) use was removed in 4831827e291e3b4a1f73679a5c1197a26d0ac3ed, so it seems like a pretty safe clean up.